tests/torch2/function_hook/nncf_graph/test_nncf_graph.py

```diff
@@ -37,6 +38,25 @@
 REF_DIR = TEST_ROOT / "torch2" / "data" / "function_hook" / "nncf_graph"
 
 
+def get_reference_graph(graph: NNCFGraph) -> nx.DiGraph:
+    out_graph = nx.DiGraph()
+    for node in sorted(graph.get_all_nodes(), key=lambda x: x.node_id):
+        attrs_node = {
+            "id": node.node_id,
+            "type": node.node_type,
+            "metatype": node.metatype.__name__,
```

Comment:
Do you have plans for updating the ` get_graph_for_structure_analysis` method? https://github.com/openvinotoolkit/nncf/blob/develop/nncf/common/graph/graph.py#L605-L643 
I believe that one method for all torch graphs is better that the current approach. I'm not stopping this PR, but please open a GH issue for that (maybe GFI?)

Reply:
Now no, This may change many times until torch2 will used as default tracing.
During this time, I dont want to change references files for each tests in nncf or some common code.